tests/fastai/test_fastai_autolog.py

```diff
@@ -94,50 +141,99 @@ def fastai_random_data_run(iris_data, fit_variant, manual_run):
 
 
 @pytest.mark.large
-@pytest.mark.parametrize("fit_variant", ["fit", "fit_one_cycle"])
-def test_fastai_autolog_logs_expected_data(fastai_random_data_run, fit_variant):
+@pytest.mark.parametrize("fit_variant", ["fit", "fit_one_cycle", "fine_tune"])
+def test_fastai_autolog_logs_expected_data(fastai_random_visual_data_run, fit_variant):
```

Comment:
Seems missing `mlflow.fastai.autolog()` line for this test. i think this will cause singlely run this test fail, like `pytest tests/fastai/test_fastai_autolog.py -k test_fastai_autolog_logs_expected_data`

Reply:
`mlflow.fastai.autolog` is called in the `fastai_random_visual_data_run` fixture.